commands/push.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const cli = require('heroku-cli-util');
+const co = require('co');
+const child = require('child_process');
+
+module.exports = function(topic) {
+  return {
+    topic: topic,
+    command: 'push',
+    description: 'Deploys a Docker image to your Heroku app',
+    needsApp: true,
+    needsAuth: true,
+    args: [{ name: 'process', optional: true }],
+    run: cli.command(co.wrap(release))
+  };
+};
+
+function* release(context, heroku) {
```

Comment:
Ha good catch. I'll also add to the help that is builds and then pushes the image to your app, for clarity.